subsys/bluetooth/host/gatt.c

```diff
@@ -180,13 +183,129 @@ BT_GATT_SERVICE_DEFINE(_2_gap_svc,
 #endif
 );
 
-#if defined(CONFIG_BT_GATT_DYNAMIC_DB)
-static void sc_ccc_cfg_changed(const struct bt_gatt_attr *attr,
-			       u16_t value)
+#if defined(CONFIG_BT_GATT_SERVICE_CHANGED)
+struct gatt_sc_cfg {
+	u8_t                    id;
+	bt_addr_le_t		peer;
+	u8_t			data[4] __aligned(4);
```

Comment:
I guess this is only going to be use for start and end range so I don't think we need to keep using an opaque storage.

Reply:
Good idea, fixed in latest push.